quantum/nodes/shors/shors.html

```diff
@@ -0,0 +1,32 @@
+<script type="text/javascript">
+  RED.nodes.registerType("shors", {
+    category: "quantum algorithms",
+    color: "#737177",
+    defaults: {
+      name: { value: "" },
+      integer: { value: 0, validate: RED.validators.number() },
+    },
+    inputs: 1,
+    outputs: 1,
+    icon: "font-awesome/fa-search",
+    align: "right",
+    paletteLabel: "Shors",
+    label: function () {
+      return this.name || "Shors";
```

Comment:
```suggestion
    paletteLabel: "shors",
    label: function () {
      return this.name || "shors";
```
For consistency, these should be lowercase.